.travis.yml

```diff
@@ -0,0 +1,41 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  global:
+    - LUAJIT_LIB=/usr/lib64/libluajit-5.1.so
+    - LUAJIT_INC=/usr/include/luajit-2.0
+  matrix:
+    - NGINX_VERSION=1.9.15
+    - NGINX_VERSION=1.8.1
+    
+before_install:
+  - sudo apt-get install -qq -y cpanminus libluajit-5.1-dev libgd-dev
+  - sudo cpanm -v --notest Test::Nginx > build.log 2>&1 || cat build.log
+  - sudo cpanm -v --notest Test::Nginx::Socket > build.log 2>&1 || cat build.log
+  
+install:
+  - wget http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz && tar -xzf nginx-${NGINX_VERSION}.tar.gz
+  - git clone https://github.com/simpl/ngx_devel_kit.git
+  - git clone https://github.com/openresty/set-misc-nginx-module.git
+  - git clone https://github.com/openresty/xss-nginx-module.git
+  - git clone https://github.com/openresty/rds-json-nginx-module.git
+  - git clone https://github.com/openresty/headers-more-nginx-module.git
+  - git clone https://github.com/openresty/lua-nginx-module.git
+  - git clone https://github.com/openresty/nginx-eval-module.git
+    
+script:
```

Comment:
@chipitsine You'll also need to clone the openresty/openresty github repos since `ngx-build` applies the patches for the nginx core (you may need to downgrade the current nginx core versions since the patches only support specific versions).